src/track/track.h

```diff
@@ -68,8 +67,8 @@ class Track : public QObject {
     Q_PROPERTY(QString durationFormattedCentiseconds READ getDurationTextCentiseconds STORED false)
     Q_PROPERTY(QString durationFormattedMilliseconds READ getDurationTextMilliseconds STORED false)
 
-    QFileInfo getFileInfo() const {
-        // Copying a QFileInfo is thread-safe (implicit sharing), no locking needed.
+    TrackFile getFileInfo() const {
```

Comment:
This one is confusing for experienced Qt users. One expects that a QFileInfo is returned. 
getTrackFile() ?

Reply:
This is for backward compatibility and intended. TrackFile has in implicit conversion to QFileInfo.

TrackFile could be named TrackFileInfo, but names should not get too long.

Reply:
Renaming the function to getFile() will cause some noise in dependent code. I will check how much.